extensions/wikia/Recirculation/WikiRecommendations.class.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+class WikiRecommendations {
+
+	const THUMBNAIL_WIDTH = 100;
+	const THUMBNAIL_RATIO = 16 / 9;
+
+	const RECOMMENDATIONS = [
+		'en' => [
+			[
+				'thumbnailUrl' => 'https://vignette1.wikia.nocookie.net/gameofthrones/images/3/3a/WhiteWalker_%28Hardhome%29.jpg/revision/latest?cb=20150601151110',
+				'title' => 'Game of Thrones',
+				'url' => 'http://gameofthrones.wikia.com/wiki/Game_of_Thrones_Wiki',
+			],
+			[
+				'thumbnailUrl' => 'https://vignette2.wikia.nocookie.net/deathnote/images/1/1d/Light_Holding_Death_Note.png/revision/latest?cb=20120525180447',
+				'title' => 'Death Note',
+				'url' => 'http://deathnote.wikia.com/wiki/Main_Page',
+			],
+			[
+				'thumbnailUrl' => 'https://vignette2.wikia.nocookie.net/midnight-texas/images/b/b0/Blinded_by_the_Light_106-01-Rev-Sheehan-Davy-Deputy.jpg/revision/latest?cb=20170820185915',
+				'title' => 'Midnight Texas',
+				'url' => 'http://midnight-texas.wikia.com/wiki/Midnight,_Texas_Wikia',
+			],
+		],
+		'de' => [],
+		'fr' => [],
+		'es' => [],
+		'pt-br' => [],
+		'ru' => [],
+		'it' => [],
+		'pl' => [],
+		'zh' => [],
+		'ja' => [],
+	];
+
+	const DEV_RECOMMENDATIONS = [
+		[
+			'thumbnailUrl' => 'https://vignette.wikia-dev.pl/gameofthrones/images/3/3a/WhiteWalker_%28Hardhome%29.jpg/revision/latest?cb=20150601151110',
+			'title' => 'Game of Thrones',
+			'url' => 'http://gameofthrones.wikia.com/wiki/Game_of_Thrones_Wiki',
+		],
+		[
+			'thumbnailUrl' => 'https://vignette.wikia-dev.pl/deathnote/images/1/1d/Light_Holding_Death_Note.png/revision/latest?cb=20120525180447',
+			'title' => 'Death Note',
+			'url' => 'http://deathnote.wikia.com/wiki/Main_Page',
+		],
+		[
+			'thumbnailUrl' => 'https://vignette.wikia-dev.pl/midnight-texas/images/b/b0/Blinded_by_the_Light_106-01-Rev-Sheehan-Davy-Deputy.jpg/revision/latest?cb=20170820185915',
+			'title' => 'Midnight Texas',
+			'url' => 'http://midnight-texas.wikia.com/wiki/Midnight,_Texas_Wikia',
+		]
+	];
+
+	const LANGUAGES = [ 'en', 'de', 'fr', 'es', 'pt-br', 'ru', 'it', 'pl', 'zh', 'ja' ];
+
+	public static function getRecommendations( $contentLanguage ) {
+		global $wgDevelEnvironment;
+
+
+		if ( empty( $wgDevelEnvironment ) ) {
+			$recommendations = self::RECOMMENDATIONS['en'];
+
+			if ( in_array( $contentLanguage, self::LANGUAGES ) ) {
+				$recommendations = self::RECOMMENDATIONS[$contentLanguage];
+			}
+			shuffle( $recommendations );
+		} else {
+			$recommendations = self::DEV_RECOMMENDATIONS;
+		}
+
+		$recommendations = array_slice( $recommendations, 0, 3 );
```

Comment:
do we need it if recommendations are hardcoded? if we still need it for some reason, can we extract limit to const?